### PR TITLE
tiflash: Update tiflash note of importing data by Lightning

### PR DESCRIPTION
**Pull request by JaySon-Huang**
Signed-off-by: JaySon-Huang <tshent@qq.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Update tiflash note about importing data by Lightning. Users can create TiFlash replica before importing data by using TiDB Lightning since v4.0.6 

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/4516
- Other reference link(s): https://github.com/pingcap/tidb-lightning/issues/273, https://github.com/pingcap/tidb-lightning/issues/391

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by JaySon-Huang** [comment]
@cyliu0  @zanmato1984 PTAL

**Review by cyliu0 — APPROVED** [review]
LGTM

File: tiflash/use-tiflash.md [hunk 1]
```diff
@@ -54,7 +54,9 @@ ALTER TABLE `tpch50`.`lineitem` SET TIFLASH REPLICA 0
     CREATE TABLE table_name like t
     ```
 
-* 目前版本里，若先对表创建 TiFlash 副本，再使用 TiDB Lightning 导入数据，会导致数据导入失败。需要在使用 TiDB Lightning 成功导入数据至表后，再对相应的表创建 TiFlash 副本。
+* 在低于 v4.0.6 的版本里，若先对表创建 TiFlash 副本，再使用 TiDB Lightning 导入数据，会导致数据导入失败。需要在使用 TiDB Lightning 成功导入数据至表后，再对相应的表创建 TiFlash 副本。
```
Comment (TomShawn) [inline tiflash/use-tiflash.md:57]:
```suggestion
* 当集群版本低于 v4.0.6，若先对表创建 TiFlash 副本，再使用 TiDB Lightning 导入数据，会导致数据导入失败。需要在使用 TiDB Lightning 成功导入数据至表后，再对相应的表创建 TiFlash 副本。
```

File: tiflash/use-tiflash.md [hunk 2]
```diff
@@ -54,7 +54,9 @@ ALTER TABLE `tpch50`.`lineitem` SET TIFLASH REPLICA 0
...
+
+* 如果集群版本以及 TiDB Lightning 版本均不低于 v4.0.6 版本，可以在表已经有 TiFlash 副本的情况下使用 TiDB Lightning 导入数据。但注意此情况会导致 TiDB Lightning 导入数据耗费的时间延长，具体取决于 TiDB Lightning 部署机器的网卡、 TiFlash 节点的 CPU 及磁盘负载、TiFlash 副本数等因素。
```
Comment (TomShawn) [inline tiflash/use-tiflash.md:59]:
```suggestion
* 如果集群版本以及 TiDB Lightning 版本均不低于 v4.0.6，可以在表已经有 TiFlash 副本的情况下使用 TiDB Lightning 导入数据。但注意此情况会导致 TiDB Lightning 导入数据耗费的时间延长，具体取决于 TiDB Lightning 部署机器的网卡、TiFlash 节点的 CPU 及磁盘负载、TiFlash 副本数等因素。
```

File: tiflash/use-tiflash.md [hunk 3]
```diff
@@ -54,7 +54,9 @@ ALTER TABLE `tpch50`.`lineitem` SET TIFLASH REPLICA 0
     CREATE TABLE table_name like t
     ```
 
-* 目前版本里，若先对表创建 TiFlash 副本，再使用 TiDB Lightning 导入数据，会导致数据导入失败。需要在使用 TiDB Lightning 成功导入数据至表后，再对相应的表创建 TiFlash 副本。
+* 如果集群版本 \< v4.0.6，若先对表创建 TiFlash 副本，再使用 TiDB Lightning 导入数据，会导致数据导入失败。需要在使用 TiDB Lightning 成功导入数据至表后，再对相应的表创建 TiFlash 副本。
+
+* 如果集群版本以及 TiDB Lightning 版本均 \>= v4.0.6，你可以在表已经有 TiFlash 副本的情况下使用 TiDB Lightning 导入数据。但注意此情况会导致 TiDB Lightning 导入数据耗费的时间延长，具体取决于 TiDB Lightning 部署机器的网卡、TiFlash 节点的 CPU 及磁盘负载、TiFlash 副本数等因素。
```
Comment (JaySon-Huang) [inline tiflash/use-tiflash.md:59]:
```suggestion
* 如果集群版本以及 TiDB Lightning 版本均 \>= v4.0.6，无论表是否已经创建 TiFlash 副本，你均可以使用 TiDB Lightning 导入数据。但注意此情况会导致 TiDB Lightning 导入数据耗费的时间延长，具体取决于 TiDB Lightning 部署机器的网卡带宽、TiFlash 节点的 CPU 及磁盘负载、TiFlash 副本数等因素。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**